### PR TITLE
add green button to homepage [#163601711]

### PR DESCRIPTION
**Pull request by NateJson**
I can add this feature using css flexbox.

+ [ ] write index.html
+ [ ] write style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>
+    Document
+  </title>
+  <link href="style.css" rel="stylesheet" />
+</head>
+<body>
+  <button class="hulk">Easy</button>
```
Comment (PuffGarbo) [inline index.html:10]: Should be an input element. Fix that

Comment (NateJson) [inline index.html:10]: should be an input element

File: style.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  justify-content: center;
+  height: 100vh;
+  margin: 0;
+}
+
+.hulk {
+  display: inline-block;
+  padding: 50px;
+  margin: 100px;
+  background-color: green;
```
Comment (PuffGarbo) [inline style.css:13]: Pick a better green

File: index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>
+    Document
```
Comment (PuffGarbo) [inline index.html:5]: Please give it a real title

File: index.html (hunk 1 above)
Comment (NateJson) [reply index.html:10]: no, CanIUse suggests that a button has a 98% use

**Review by PuffGarbo — COMMENTED** [review]
good first try. Give it another go

**Review by NateJson — COMMENTED** [review]
@jacobcov please review asap

**Pull request merged**